tests/cache/tests.py

```diff
@@ -2162,6 +2162,31 @@ def test_304_response_has_http_caching_headers_but_not_cached(self):
         self.assertIn('Cache-Control', response)
         self.assertIn('Expires', response)
 
+    def test_cache_control_max_age(self):
+        view = cache_page(2)(hello_world_view)
+        request = self.factory.get('/view/')
+
+        # Request once
+        with mock.patch.object(time, 'time', return_value=1468749600):  # Sun, 17 Jul 2016 10:00:00 GMT
+            response = view(request)
+            response.close()
+            self.assertIn('Expires', response)
+            self.assertEqual('Sun, 17 Jul 2016 10:00:02 GMT', response['Expires'])
+            self.assertIn('Cache-Control', response)
+            self.assertEqual('max-age=2', response['Cache-Control'])
+            self.assertNotIn('Age', response)
+
+        # Response for second request made later must have Age header
```

Comment:
Could you add comments that describe the calculations that lead the expected Age value?